### PR TITLE
fix(slack): add support of blank space in handlebar

### PR DESCRIPTION
**Pull request by ItsSudip**
## What are the changes introduced in this PR?

If any property key is some string with spaces then we need to cover that string with quotes(") inside the handlebar expression. Otherwise, it will throw an error. So with this PR, we are fixing that issue.

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Type of change

If the pull request is a **bug-fix**, **enhancement** or a **refactor**, please fill in the details on the changes made.

- Existing capabilities/behavior

- New capabilities/behavior

If the pull request is a **new feature**,

### Any technical or performance related pointers to consider with the change?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### If the PR has changes in more than 10 files, please mention why the changes were not split into multiple PRs.

N/A

### If multiple linear tasks are associated with the PR changes, please elaborate on the reason:

N/A

<hr>

### Developer checklist

- [x] **No breaking changes are being introduced.**

- [ ] Are all related docs linked with the PR?

- [x] Are all changes manually tested?

- [ ] Does this change require any documentation changes?

- [x] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

File: test/integrations/destinations/slack/processor/data.ts
Comment (krishna2020) [inline test/integrations/destinations/slack/processor/data.ts:1]: Is this only formatting changes or the test case for key with space has been added ?
Also, can you update the test case descriptions looping in @anantjain45823 ?

Comment (anantjain45823) [reply test/integrations/destinations/slack/processor/data.ts:1]: For test cases description I will take care

Comment (ItsSudip) [reply test/integrations/destinations/slack/processor/data.ts:1]: I have added test case as well.

Comment (anantjain45823) [reply test/integrations/destinations/slack/processor/data.ts:1]: @ItsSudip I have added the test cases description can you update the description for the ones you have updated

**Review by anantjain45823 — APPROVED** [review]

**Review by shrouti1507 — APPROVED** [review]

**Review by yashasvibajpai — APPROVED** [review]

**Review by sanpj2292 — APPROVED** [review]

**Pull request merged**